art.taunoerik.tauno-serial-plotter.yml

```diff
@@ -0,0 +1,37 @@
+app-id: art.taunoerik.tauno-serial-plotter
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
```

Comment:
does this need the KDE runtime? I don't see PyQt being build to use Qt.

Reply:
Yes, it does. It is in a separate file "python3-requirements.json". I am not exactly sure how to add it.

Reply:
ah yeah sorry I totallky missed it. All good.

Reply:
I will add appdata file.